population/dimensions_test.go

```diff
@@ -15,6 +15,87 @@ import (
 	. "github.com/smartystreets/goconvey/convey"
 )
 
+func TestGetDimensionCategories(t *testing.T) {
+	const userAuthToken = "user"
+	const serviceAuthToken = "service"
+	const populationType = "UR"
+	const dimension = "sex"
+
+	Convey("Given a valid request", t, func() {
+		stubClient := newStubClient(&http.Response{Body: ioutil.NopCloser(bytes.NewReader(nil))}, nil)
+		client, err := NewWithHealthClient(health.NewClientWithClienter("", "http://test.test:2000/v1", stubClient))
+		So(err, ShouldBeNil)
+		input := GetDimensionCategoryInput{
+			AuthTokens: AuthTokens{
+				UserAuthToken:    userAuthToken,
+				ServiceAuthToken: serviceAuthToken,
+			},
+			PopulationType: populationType,
+			Dimensions:     []string{dimension},
```

Comment:
Probably best to test with more than one dimension to be sure the comma separation is good

Reply:
added a test with 2 dimensions

Reply:
this has been added